web-client/src/ducks/requests/functions.ts

```diff
@@ -51,20 +51,28 @@ export const setUserRequest = async ({
     .withConverter(RequestFirestoreConverter)
     .set(requestPayload);
 
-export const getOpenPost = async ({ lat, lng }: IgetRequestPosts) =>
+export const getCavRequestPosts = async ({ lat, lng }: IgetRequestPosts) =>
   functions.httpsCallable('https-api-requests-getRequests')({
     lat,
     lng,
-    radius: 500,
-    status: AbstractRequestStatus.pending,
+    radius: 5,
+    status: AbstractRequestStatus.accepted,
   });
 
-export const getAcceptedPost = async ({ lat, lng }: IgetRequestPosts) =>
+export const getFindPosts = async ({ lat, lng }: IgetRequestPosts) =>
   functions.httpsCallable('https-api-requests-getRequests')({
```

Comment:
Can you switch to using firestore directly?

Reply:
In my next pull request.